ui/src/components/core/core-modal.ts

```diff
@@ -51,6 +53,9 @@ export class CoreModal extends LitElement {
   @property({type: Boolean})
   accessor isPersistent = true;
 
+  @property(({type: Boolean, reflect: true}))
```

Comment:
So far, I always changed the reflected attribute to more closely resemble a normal HTML attribute. We haven't discussed this yet, so maybe we should talk about this sometime?
```suggestion
  @property(({type: Boolean, attribute: 'no-padding', reflect: true}))
```

Reply:
When do we use a '-' an when not? For example, 'isactive' is used without

Reply:
I would:
- drop `is` or `has` prefix, if present
- insert `-` before each uppercase character
So `isactive` should probably be `active`.